### DISCUSSION TITLE
Wicked Engine

### DISCUSSION THREAD

**Discussion by socialtwisty** [jrouwe/JoltPhysics#1268] [General]
Hey there! I wanted to bring attention to Wicked Engine ( https://wickedengine.net https://discord.gg/CFjRYmE ) as a candidate for the "Projects Using Jolt" list.

The author switched from Bullet to Jolt earlier this year and in my opinion perform wonderfully.  Rigid and softbody support has been added along with ragdolls for characters.

I think you'll agree its a visually impressive ECS based framework and easy to work with even for those like me with limited experience outside of Editor based engines like Godot, Unreal & Unity.

My hope and plea in bringing attention to this engine is that it might interest someone with experience to help with the inclusion of rigid body constraints as they sadly are not exposed engine-side yet. With tutorials scarce, I'm actively working on an open pinball game for newbies like me to learn from, and it would be amazing to implement things like point and hinge based constraints.

**Comment by jrouwe** [comment]
Done.

**Reply by socialtwisty** [reply]
Thank you so much! :D
